examples/EagerMode/QuickStart/utils.py

```diff
@@ -0,0 +1,14 @@
+import numpy as np
+
+
+def calc_discount_norm_reward(reward_list, gamma):
```

Comment:
Could you add some comment for this function and the one used in QuickStart folder?